### ISSUE TITLE
Health check always waits check_ttl before first check

### ISSUE THREAD

**Issue by travigd** [vmagamedov/grpclib#105]
A side effect of the current implementation of `ServiceCheck.__check__`

https://github.com/vmagamedov/grpclib/blob/9008ca89e237502678eba336a5c13fbf6a40efb0/grpclib/health/check.py#L94-L96

is that we will always wait at least `check_ttl` seconds before we run the first check. Is this desired behavior? From my perspective it is not.

The workaround for this is to set `check_ttl` equal to zero (and optionally do your own caching for the health check value).

One way to fix this would be to set `self._last_check = -(check_ttl + 1)` so that the condition always passes, or to initialize it to `None` and write something like:
```python3
# Add test for self._last_check, always run check if it is None (or even 0)
if self._last_check and time.monotonic() - self._last_check < self._check_ttl:
    return self._value
```

**Comment by vmagamedov** [comment]
This behaviour wasn't desired. The problem is that I was unaware that `time.monotonic()` can start from `0`. I just checked that you can get big initial values in Linux (looks like a system's uptime) and close to `0` in macOS (looks like a process lifetime). Maybe it also depends on a Python version. And it was simply wrong to make any assumptions.

So this is definitely a bug. Thank you for this finding.

**Comment by vmagamedov** [comment]
Fixed in 0c20bb57ed167f71f4954b99e0bff043c0cba1cd
